lib/src/cli/flutter_cli.dart

```diff
@@ -493,22 +515,6 @@ String? _topGroupName(Test test, Map<int, TestGroup> groups) => test.groupIDs
     .map((groupID) => groups[groupID]?.name)
     .firstWhereOrNull((groupName) => groupName?.isNotEmpty ?? false);
 
-String _actualTestPath(
-  bool optimizationApplied,
-  String path,
-  String? groupName,
-) =>
-    optimizationApplied
-        ? path.replaceFirst(_testOptimizerFileName, groupName!)
-        : path;
-
-String _actualTestName(
-  bool optimizationApplied,
-  String name,
-  String? topGroupName,
-) =>
-    optimizationApplied ? name.replaceFirst(topGroupName!, '').trim() : name;
-
```

Comment:
What's the reasoning behind removing these functions? I see that now we do one less check of `optimizationApplied` is this why?